packages/flutter/lib/src/widgets/container.dart

```diff
@@ -371,71 +371,86 @@ class Container extends StatelessWidget {
   /// method throws an [UnsupportedError].)
   final Clip clipBehavior;
 
+  @override
+  State<Container> createState() => _ContainerState();
+}
+
+class _ContainerState extends State<Container> {
+  final GlobalKey _childGlobalKey = GlobalKey();
```

Comment:
This also might be a performance concern. Somewhat related: https://github.com/flutter/flutter/issues/161698#issuecomment-2596845482